application.py

```diff
@@ -148,3 +168,22 @@ def show_qr_code(self):
                 self.logs["app.show_qr_code"] = "QR-код и сообщение успешно отображены."
         except Exception as e:
             self.logs["app.show_qr_code"] = f"Ошибка при отображении QR-кода и сообщения: {e}"
+
+    def keyPressEvent(self, qKeyEvent):
+        if qKeyEvent.key() == Qt.Key_Space: 
+            self.openMenu()
+        else:
+            super().keyPressEvent(qKeyEvent)
+    
+    def openMenu(self):
+        if not self.menu_window:
+            self.menu_window = Menu()
+            self.menu_window.sig.connect(self.menuClosed) # type: ignore
+
+            self.menu_window.show()
+        else:
+            self.menu_window.hide()
+            self.menu_window = None
```

Comment:
Why do you recreate and delete it each time? I am not 100% sure, but by doing so you might be leaking memory.

Reply:
✅